### PR TITLE
Updating charts readme

### PR DESCRIPTION
**Pull request by ostromart**
This is a first pass to steer users to everything they need to do when making changes to the manifests dir. Lots of opportunity for process improvement here :-) At least this is a start. 
This needs to be rebased once https://github.com/istio/istio/pull/20218 merges, in the meantime, please ignore everything except the markdown files.

### CODE REVIEW

File: manifests/UPDATING-CHARTS.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+If any values.yaml changes are being made, the same changes must be made in the `operator/data/profiles/default.yaml`
+file, which must be in sync with the Helm values in `manifests`.
+
+### Step 3. Update the validation schema
+
+Istioctl uses a [schema](../operator/pkg/apis/istio/v1alpha1/values_types.proto) to validate the values. Any changes to
+the schema must be added here, otherwise istioctl users will see errors. 
+Once the schema file is updated, run:
+```bash
+$ make gen-operator-proto
```
Comment (howardjohn) [inline manifests/UPDATING-CHARTS.md]: maybe suggest build_with_container=1 until we get that as the default

**Review by howardjohn — COMMENTED** [review]
I think you picked up extra changes in here. README lgtm

File: manifests/UPDATING-CHARTS.md (hunk 1 above)
Comment (ostromart) [reply manifests/UPDATING-CHARTS.md]: i was thinking the make command would include that. this is a target that it's never really possible to do a local build.

**Comment by howardjohn** [comment]
I am not sure what you mean? in this repo container build is off by
default. Or are users expected to run these commands from operator/?

We should have all make targets in the top level I think

**Comment by ostromart** [comment]
i mean even if build container is off, we turn it on for this target. we can echo an explanation for the time being so people don't get confused.

**Comment by howardjohn** [comment]
we cannot turn it on or off selectively. I guess if you run make commands
from within operator/ it will use the build container but really that's
just because we haven't merged the two makefiles yet?

Id the targets are only in operator readme let's just make it clear you
need to be in operator/

File: manifests/UPDATING-CHARTS.md (hunk 1 above)
Comment (ostromart) [reply manifests/UPDATING-CHARTS.md]: Done.

**Review by howardjohn — APPROVED** [review]

**Comment by howardjohn** [comment]
Can you rebase? This is blocking devs who don't know how to submit their changes

**Comment by howardjohn** [comment]
can we rebase this?

**Comment by howardjohn** [comment]
@ostromart any plans to rebase? it would be pretty handy to point folks to this

**Review by fpesce — APPROVED** [review]
:shipit:

**Pull request merged**